public/components/datasources/components/manage/accelerations/create_accelerations_flyout/create/create_acceleration.tsx

```diff
@@ -45,6 +45,7 @@ export interface CreateAccelerationProps {
   resetFlyout: () => void;
   databaseName?: string;
   tableName?: string;
+  dataSourceClientId?: string;
```

Comment:
curious is this the id of the data source client, or the id of the data source? i.e. would the id be used for something else for the data source other than client?